Tests/UI/LambdaTestPlayground/TestSpecialPage.spec.js

```diff
@@ -0,0 +1,71 @@
+const { test, expect } = require('@playwright/test');
+const { SpecialMenu } = require('../../../../PlaywrightJS/Pages/LambdaTestPlayground/SpecialsPage');
+const { Cameras } = require('../../../../PlaywrightJS/Pages/LambdaTestPlayground/CamerasPage');
+const { runTest } = require('../../SetupTest');
+
+test('Verify the primary headers are displayed on the Special page', runTest(async ({ page }) => {
+
+  const specialMenu = new SpecialMenu(page)
+  const websiteLink = 'https://ecommerce-playground.lambdatest.io/';
+  await page.goto(`${websiteLink}`);
+  await specialMenu.clickSpecialTab();
+  
+  // Headers on sidebar left side
+  expect(await specialMenu.isSpecialOffersHeaderVisible()).toBe(true);
+  expect(await specialMenu.allsidebarHeadersDisplayed('Filter')).toBe(true);
+  expect(await specialMenu.allsidebarHeadersDisplayed('Price')).toBe(true);
+  expect(await specialMenu.allsidebarHeadersDisplayed('Search')).toBe(true);
+  expect(await specialMenu.allsidebarHeadersDisplayed('Availability')).toBe(true);
+  expect(await specialMenu.allsidebarHeadersDisplayed('Discount')).toBe(true);
+  expect(await specialMenu.allsidebarHeadersDisplayed('Rating')).toBe(true);
+  expect(await specialMenu.allsidebarHeadersDisplayed('Desktops')).toBe(true);
+  expect(await specialMenu.allsidebarHeadersDisplayed('Laptops')).toBe(true);
+  expect(await specialMenu.allsidebarHeadersDisplayed('Components')).toBe(true);
+  expect(await specialMenu.allsidebarHeadersDisplayed('Tablets')).toBe(true);
+  expect(await specialMenu.allsidebarHeadersDisplayed('Software')).toBe(true);
+  expect(await specialMenu.allsidebarHeadersDisplayed('Phones & PDAs')).toBe(true);
+  expect(await specialMenu.allsidebarHeadersDisplayed('Cameras')).toBe(true);
+  expect(await specialMenu.allsidebarHeadersDisplayed('MP3 Players')).toBe(true);
+  expect(await specialMenu.isMinimumPriceFieldVisible()).toBe(true);
+  expect(await specialMenu.isMaximumPriceFieldVisible()).toBe(true);
+  expect(await specialMenu.allCollapsingArrowsVisible()).toBe(true);
+
+  // Other headers in middle of page
+  expect(await specialMenu.isGridViewIconVisible()).toBeTruthy();
+  expect(await specialMenu.isListViewIconVisible()).toBeTruthy();
+  const productCompareLink = await specialMenu.isProductCompareLinkVisible();
+  console.log(`Product Compare link visible? ${productCompareLink}`);
```

Comment:
change this and put it in the assertion on like 38

Reply:
What did you mean by this one again? Am I supposed to remove something or move?